### PR TITLE
refactor: split logger config and service

### PR DESCRIPTION
**Pull request by yangby-cryptape**
_Originally posted by @doitian in https://github.com/nervosnetwork/ckb/pull/2199#discussion_r465498134_

- Split logger config and service.
- Let `ckb-app-config` only be dependent on `ckb-logger-config`.
- Move the function which checks identifier into `ckb-util`.

### CODE REVIEW

File: util/app-config/Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ serde_plain = "0.3.0"
 toml = "0.5"
 path-clean = "0.1.0"
 ckb-logger = { path = "../../util/logger" }
-ckb-logger-service = { path = "../../util/logger-service" }
+ckb-logger-config = { path = "../../util/logger-config" }
 sentry = "0.16.0"
```
Comment (quake) [inline util/app-config/Cargo.toml:16]: useless dependencies?

Comment (yangby-cryptape) [reply util/app-config/Cargo.toml:16]: No.

Useless dependencies will be found by [`devtools/ci/check-cargotoml.sh`](https://github.com/nervosnetwork/ckb/blob/057b7e2/devtools/ci/check-cargotoml.sh) and CI will be failed.

**Comment by yangby-cryptape** [comment]
CI is blocked, "force-push" to re-run it.

**Comment by keroro520** [comment]
bors r=TheWaWaR,doitian,keroro520

**Pull request merged**